### PR TITLE
add test

### PR DESCRIPTION
**Pull request by maverick-peng**
gitflow test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,162 +1,174 @@
...
 2. Angular
-	* Component
-	* Service
-	* Module
-	* Routing
-	* Directive
-	* HttpClient
-	* ReactiveForm
-	* Pipe
+
```
Comment (josephjsf2) [inline README.md:6]: test

File: README.md [hunk 2]
```diff
@@ -1,162 +1,174 @@
...
-* 登入失敗時，需顯示錯誤訊息
-* 當欄位被使用者點過且未輸入資料時，需要在欄位下方顯示『XX欄位不可留空』
-* 只要輸入欄位未輸入完整資料，則登入按鈕不可被點擊
-* 輸入完密碼後，實做 peek 功能，讓使用者可以看到自己輸入的密碼
-* 需防止使用者 Double click登入按鈕
-* 登入成功後，將畫面重導至會員帳號清單功能(/pages/member)
-* 只要使用者未登入，統一將瀏覽器重導回登入頁
-* 若使用者已登入過，則不再進入登入畫面(透過瀏覽器連結觸發)，統一導至 /pages/member
+
+### 1. 登入頁 - 需要在登入後從 API 取得 authToken 資訊
```
Comment (josephjsf2) [inline README.md:46]: 1234565

**Review by josephjsf2 — CHANGES_REQUESTED** [review]
change it

File: README.md (hunk 2 above)
Comment (josephjsf2) [reply README.md:46]: 21245

**Comment by maverick-peng** [comment]
> change it
ok

**Review by josephjsf2 — APPROVED** [review]
ok

**Pull request merged**